src/pages/Home.tsx

```diff
@@ -1,28 +1,41 @@
 //TODO (UI / UX) - These icons are not final
-import { Toolbar, Typography } from '@mui/material';
+import {
+    Box,
+    Collapse,
+    IconButton,
+    Toolbar,
+    Typography,
+    useTheme,
+} from '@mui/material';
 import { authenticatedRoutes } from 'app/routes';
+import Dashboard from 'components/dashboard';
 import HeroImageAndDescription from 'components/hero';
 import LoginNotifications from 'components/login/Notifications';
+import { useShowDashboardWelcome } from 'context/DashboardWelcome';
+import { defaultOutline, semiTransparentBackground } from 'context/Theme';
 import useGlobalSearchParams, {
     GlobalSearchParams,
 } from 'hooks/searchParams/useGlobalSearchParams';
 import usePageTitle from 'hooks/usePageTitle';
+import { Plus, Xmark } from 'iconoir-react';
 import { useMemo } from 'react';
 import { FormattedMessage } from 'react-intl';
 
 const Home = () => {
-    usePageTitle({ header: authenticatedRoutes.home.title });
+    const theme = useTheme();
```

Comment:
Since we are expanding the functionality of this page... we should now toss the error message into a stand along component. Since the search param stuff usually causes extra renders.

![image](https://github.com/user-attachments/assets/81f97e4c-14eb-4320-b8c7-682e1f19b3e9)